manifests/init.pp

```diff
@@ -267,7 +271,9 @@
       require      => Class['certs::apache'],
     }
 
-    include foreman_proxy_content::pub_dir
+    class { 'foreman_proxy_content::pub_dir':
+      pub_dir_options => $pub_dir_options,
```

Comment:
Looking at https://github.com/theforeman/puppet-foreman_proxy_content/blob/master/manifests/pub_dir.pp#L12 it looks like this is not actually used anywhere?

The options that `apache::vhost` receives seem to be hard coded at https://github.com/theforeman/puppet-foreman_proxy_content/blob/master/manifests/pub_dir.pp#L24

Reply:
This is used in the template: https://github.com/theforeman/puppet-foreman_proxy_content/blob/master/templates/httpd_pub.erb#L7